pkgs/development/interpreters/python/build-python-package-common.nix

```diff
@@ -10,7 +9,7 @@
 , ... } @ attrs:
 
 attrs // {
-  buildInputs = buildInputs ++ [ bootstrapped-pip ];
+  buildInputs = buildInputs ++ [ python.pythonForBuild.pkgs.bootstrapped-pip ];
```

Comment:
Bootstrapped pip is used at build time and so should be a `nativeBuildInput`.

Reply:
Alternatively, since you use it explicitly below, maybe it doesn't need to be any sort of imput.

Reply:
Good catch. I don't think we actually need it as a build input. We call it already from the store path, and Python packages should never call or import from pip. Doesn't mean they won't do it though...